src_c/draw.c

```diff
@@ -850,6 +850,199 @@ polygon(PyObject *self, PyObject *arg, PyObject *kwargs)
         return pgRect_New4(l, t, 0, 0);
 }
 
+#define POLYGONS_ERR_INSUFFICIENT_ARGS 0
+#define POLYGONS_ERR_WRONG_SURF_FORMAT 1
+#define POLYGONS_ERR_SEQUENCE_REQUIRED 2
+#define POLYGONS_ERR_POINTS_SEQUENCE_REQUIRED 3
+#define POLYGONS_ERR_POINTS_INCORRECT_LENGTH 4
+#define POLYGONS_ERR_ALLOCATION 5
+#define POLYGONS_ERR_LOCK 6
+#define POLYGONS_ERR_UNLOCK 7
+
+static PyObject *
+polygons(PyObject *self, PyObject *const *args, Py_ssize_t nargs)
+{
+    pgSurfaceObject *surfobj;
+    PyObject *colorobj, *points, *item = NULL, *draw_sequence;
+    PyObject *poly_repr = NULL;
+    PyObject **f_drawsequence;
+    PyObject *width;
+    SDL_Surface *surf = NULL;
+    Uint8 rgba[4];
+    Uint32 color;
+    int *xlist = NULL, *ylist = NULL;
+    int l_width = 0; /* Default width. */
+    int x, y, result, l, t;
```

Comment:
appveyor is complaining that these variables are unused.

Reply:
yeap, pushing a fix.